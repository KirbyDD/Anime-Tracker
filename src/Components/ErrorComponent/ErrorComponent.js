import React from "react";
import { Link } from "react-router-dom";
import "./ErrorComponent.scss";

export const ErrorComponent = () => {
  return (
    <section className="error-section">
      <h2 className="error-msg">
        404 Error. The page you are looking for does not exist. Please click the
        button below to go back to the Home Page
      </h2>
      <Link to="/" className="home-link">
        Naruto Run Home
      </Link>
    </section>
  );
};

export default ErrorComponent;
