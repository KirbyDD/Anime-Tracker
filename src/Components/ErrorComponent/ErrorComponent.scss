.error-section {
  font-size: 40px;
  padding: 60px;
  text-align: center;
}

.home-link {
  color: gold;
}
