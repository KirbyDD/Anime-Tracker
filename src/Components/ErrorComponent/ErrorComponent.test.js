import React from "react";
import { shallow } from "enzyme";
import { ErrorComponent } from "./ErrorComponent";

describe("Error Component", () => {
  let wrapper;

  beforeEach(function() {
    wrapper = shallow(<ErrorComponent />);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
