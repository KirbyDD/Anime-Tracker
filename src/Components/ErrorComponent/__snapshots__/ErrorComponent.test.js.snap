// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component should match snapshot 1`] = `
<section
  className="error-section"
>
  <h2
    className="error-msg"
  >
    404 Error. The page you are looking for does not exist. Please click the button below to go back to the Home Page
  </h2>
  <Link
    className="home-link"
    to="/"
  >
    Naruto Run Home
  </Link>
</section>
`;
